### DISCUSSION TITLE
Search for locations should be "fuzzy"

### DISCUSSION THREAD

**Discussion by benhammondmusic** [SatcherInstitute/health-equity-tracker#4295] [HET Ideas (Features, Requests, New Data, etc)]
## PROBLEM: 
The geography search box should implement fuzzy matches where possible. For example, to find `St. John, Virgin Islands`, any of the following searches should work:
- `St John` (no period)
- `Saint John`

Would be ideal to have aliasing as well, for locations that have extremely common alternate names:. For example, the following searches don't yield the expected results
- `USVI` 
- `DC`
- `US` or `USA`

If possible, should ignore the word "county"; right now typing `Sarasota, Fl` doesn't yield because you need to type `county` in as well

**Comment by jgonzalezmsm** [comment]
all great ideas. can we get an estimate on effort and would it make sense to include the city / county search as part of this?

**Comment by jgonzalezmsm** [comment]
Think this is still worthwhile - let's see when timing makes sense.

**Comment by benhammondmusic** [comment]
Promoted to issue #4917 in the new [Better Location Search](https://github.com/SatcherInstitute/health-equity-tracker/milestone/56) milestone, alongside city search (#4918).
